python/ray/worker.py

```diff
@@ -28,6 +28,7 @@
 import ray.remote_function
 import ray.serialization as serialization
 import ray._private.services as services
+import ray._private.package as ray_pkg
```

Comment:
```suggestion
import ray._private.runtime_env
```

How about we rename this to "runtime_env" and avoid import aliasing?